test/request_cache/plug_test.exs

```diff
@@ -0,0 +1,41 @@
+defmodule RequestCache.PlugTest do
+  @moduledoc false
+
+  use ExUnit.Case, async: false
+
+  import ExUnit.CaptureLog
+
+  defmodule FailingCacheModule do
+    def get(_cache_key) do
+      {:error, %{reason: :timeout}}
+    end
+  end
+
+  describe "call/2" do
+    @expected_log_content "reason: :timeout"
+
+    setup do
+      Application.put_env(:request_cache_plug, :request_cache_module, FailingCacheModule)
```

Comment:
We probably should never overwrite this here it changes every future test after it